src/pages/CountryInfo.module.css

```diff
@@ -0,0 +1,44 @@
+.infoContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  row-gap: 5vh;
+  padding-top: 15vh;
+  padding-bottom: 15vh;
+}
+
+.vector {
+  background-color: #5586e2;
+  height: 50px;
+  width: 50px;
+}
+
+.flag {
+  border: 1px solid black;
+}
+
+.infoList {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] Nice job adding the back button arrow to the details page, but Kindly add it at the left of the nav bar to match the given design.

- [ ] The country details elements are not centered and overflowed from the right, Kindly fix that.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/204022978-bef8ead8-fb31-4f5d-91a6-1f62b5f2e89a.png)  | ![image](https://user-images.githubusercontent.com/68796232/204022966-32ba1c5d-24c1-43ad-be12-c8b7291bad81.png)  |